bde_verify.cfg

```diff
@@ -7,6 +7,9 @@ set enterprise Bloomberg
 # Maximum number of lines and statements in an inline function.
```

Comment:
You cannot modify existing configuration in this file. Just add the required lines for the check.